docs/source/launch_parameters.md

```diff
@@ -0,0 +1,74 @@
+# Launch Parameters for Text Generation Inference
+
+Text Generation Inference allows you to customize the way you serve your models. You can use the following parameters to configure your server. You can enable them by adding them environment variables or by providing them as arguments when running `text-generation-launcher`. Environment variables are in `UPPER_CASE` and arguments are in `lower_case`.
+
+## Model
+
+- `MODEL_ID` - The name of the model to load. Can be a MODEL_ID as listed on huggingface.co like `gpt2` or `OpenAssistant/oasst-sft-1-pythia-12b`. Or it can be a local directory containing the necessary files as saved by `save_pretrained(...)` methods of transformers. Default: `bigscience/bloom-560m`.
```

Comment:
Same here, I'd feel more confortable automating the generation for this.

`text-generation-launcher --help` contains all of those, so we should be able to extract everything from here wdyt ?
To we could have something in the CI that does.


`update-doc --check` (or `update-doc && git diff --is-empty`)

Reply:
Happy to automate but key point we had with customers is they didn't understand whats possible and especially that env vars have a different naming schema. Happy to update if there is an automated way, which is helpful and good.

Reply:
@philschmid I'm currently working on it, there are two different notes redirecting folks to use `text-generation-launcher --help` for more in different places across docs. maybe we could resurface that.